File: services/reservation/src/main/java/com/poc/reservation/util/response/HospitalResponse.java
package com.poc.reservation.util.response;

import com.poc.reservation.model.MedicalSpeciality;
import lombok.*;

import java.util.Set;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class HospitalResponse {
    private Integer id;
    private String nomOrganisation;
    private String adresse;
    private String codePostal;
    private Integer litsDisponible;
    private float longitude;
    private float latitude;
    private Set<MedicalSpeciality> specialitesMedicales;
}

File: poc-medhead-frontend/src/app/services/models/hospital-response.ts
/* tslint:disable */
/* eslint-disable */
import { MedicalSpeciality } from '../models/medical-speciality';
export interface HospitalResponse {
  adresse?: string;
  codePostal?: string;
  id?: number;
  latitude?: number;
  litsDisponible?: number;
  longitude?: number;
  nomOrganisation?: string;
  specialitesMedicales?: Array<MedicalSpeciality>;
}
